src/main/java/com/shopify/model/ShopifyApplicationCharge.java

```diff
@@ -0,0 +1,41 @@
+package com.shopify.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ShopifyApplicationCharge {
+
+    @XmlElement(name = "confirmation_url")
+    String confirmationUrl;
+    @XmlElement(name = "created_at")
+    String createdAt;
```

Comment:
this will go as a string right.. can we convert these timestamp objects to DateTime as we do for other models rather than string?
```
@XmlElement(name = "created_at")
@XmlJavaTypeAdapter(DateTimeAdapter.class)
private DateTime createdAt;
```

Reply:
It was String in some classes. I will check and fix it.